client/lib/wrap-es6-functions/index.js

```diff
@@ -28,12 +28,18 @@ function wrapObjectFn( obj, objectName, key ) {
 	}
 }
 
-export default function() {
+const wrapEs6Functions = function() {
+	if ( process.env.NODE_ENV === 'development' ) {
```

Comment:
If this code stats, it will do nothing. Both env guards are mutually exclusive 😃